src/js/smooch.jsx

```diff
@@ -58,7 +58,7 @@ function renderLink() {
 function getDeviceId() {
     const SK_STORAGE = 'sk_deviceid';
     const deviceId = storage.getItem(SK_STORAGE) ||
-        uuid.v4().replace(/-/g, '');
+    uuid.v4().replace(/-/g, '');
```

Comment:
Looks like a sublime-esformatter problem.. :(

Works in command line esformatter
```
▶ <test.js
'use strict';

let blap = 42 ||
100;

function foo() {}

let bar = foo() ||
96;

~/git/agent-console  b/cox-email-marked ✗                                                                                                            11m ◒
▶ esformatter test.js
'use strict';

let blap = 42 ||
    100;

function foo() {
}

let bar = foo() ||
    96;
```